app/src/main/java/org/schabi/newpipe/util/ThemeHelper.java

```diff
@@ -96,7 +103,19 @@ public static int getDefaultTheme(Context context) {
      */
     @StyleRes
     public static int getDialogTheme(Context context) {
-        return isLightThemeSelected(context) ? R.style.LightDialogTheme : R.style.DarkDialogTheme;
+        String lightTheme = context.getResources().getString(R.string.light_theme_key);
+        String darkTheme = context.getResources().getString(R.string.dark_theme_key);
+        String blackTheme = context.getResources().getString(R.string.black_theme_key);
+        String terminalTheme = context.getResources().getString(R.string.terminal_theme_key);
```

Comment:
``` Java
String lightTheme = context.getResources().getString(R.string.light_theme_key);
String darkTheme = context.getResources().getString(R.string.dark_theme_key);
String blackTheme = context.getResources().getString(R.string.black_theme_key);
String terminalTheme = context.getResources().getString(R.string.terminal_theme_key);
```

You can find these four lines in a bunch of static methods in this class. Do you think it is possible to make create four class attributes and use them instead? I do not have time to check this out now, but this might be tricky because the context is needed to initialize them.